### PR TITLE
chore: generalize error msg to error.go

### LINKED ISSUE #758: [DISCUSS] Create a new file to generalize error response
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [x] Question or discussion
- [ ] Bug
- [ ] Requirements
- [ ] Feature or performance improvement
- [ ] Other

___
### Requirements or improvement
- Please describe your requirements or improvement suggestions.

Hi, I recommend creating a new file to generalize error response which is more convenient and normative.

Now:
https://github.com/apache/apisix-dashboard/blob/a050e8464165c973663fc14fa5a7ecfd7255f1ae/api/filter/authentication.go#L39-L42

Advice:

```go

var (
  ErrorUnauthorized = gin.H{"code": 010013, "message": "Request Unauthorized")
)
```

### PR DESCRIPTION
**Pull request by starsz**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
fix #758 

### New feature or improvement
- Describe the details and related test reports.

Move the error msg to error.go.So that we can update the error msg more easily.
___

### CODE REVIEW

**Comment by juzhiyuan** [comment]
would better request changes from members

File: api/internal/utils/consts/error.go [hunk 1]
```diff
@@ -16,13 +16,33 @@
...
+	"github.com/shiningrush/droplet/data"
+)
 
 const (
 	ErrBadRequest = 20001
 	ErrForbidden  = 20002
 )
 
+const (
+	UpstreamIDNotFound = "upstream id: %s not found"
```
Comment (nic-chen) [inline api/internal/utils/consts/error.go:31]: Could `upstream` be changed to a variable?

File: api/internal/utils/consts/error.go [hunk 2]
```diff
@@ -16,13 +16,33 @@
...
+	ServiceIDNotFound  = "service id: %s not found"
+)
+
+var (
+	ErrorUsernamePassword         = errors.New("username or password error")
```
Comment (tokers) [inline api/internal/utils/consts/error.go:36]: It's a good coding style to give each exportable variable a comment paragraph to explain its meaning.

File: api/internal/utils/consts/error.go (hunk 1 above)
Comment (starsz) [reply api/internal/utils/consts/error.go:31]: Yes. We can.

File: api/internal/utils/consts/error.go (hunk 2 above)
Comment (starsz) [reply api/internal/utils/consts/error.go:36]: Good idea.

Comment (starsz) [reply api/internal/utils/consts/error.go]: Updated.

File: api/internal/utils/consts/error.go (hunk 1 above)
Comment (starsz) [reply api/internal/utils/consts/error.go]: Updated.

**Review by nic-chen — APPROVED** [review]

**Review by Jaycean — APPROVED** [review]
LGTM

**Review by tokers — APPROVED** [review]

**Review by gxthrj — APPROVED** [review]

**Pull request merged**